README.md

```diff
@@ -1,13 +1,13 @@
 Twitch API | Rust library for talking with the new Twitch API aka. "Helix", TMI and more!
 ============================================
 
-[![github]](https://github.com/twitch-rs/twitch_api)&ensp;[![crates-io]](https://crates.io/crates/twitch_api2)&ensp;[![docs-rs-big]](https://docs.rs/twitch_api2/0.6.1/twitch_api)
+[![github]](https://github.com/twitch-rs/twitch_api)&ensp;[![crates-io]](https://docs.rs/twitch_api2)&ensp;[![docs-rs-big]](https://docs.rs/twitch_api2)
```

Comment:
Oops, missed this. Thanks for spotting it.

Id like to keep the version number here, if you could fix that and fixup the commit ill merge afterwards. 👍

Reply:
also the crates.io link should stay

Reply:
I THINK it is done, but im not sure because i had to make the commit using a cellphone.